code/datums/components/aura_healing.dm

```diff
@@ -0,0 +1,147 @@
+#define HEAL_EFFECT_COOLDOWN (1 SECONDS)
```

Comment:
shouldnt this be configurable from AddComponent

Reply:
This is only the visual effect, it doesn't matter that much